docs/09-Networking/19-DNS-in-kubernetes.md

```diff
@@ -26,7 +26,7 @@ $ kubectl create ns apps
 $ kubectl run nginx --image=nginx --namespace apps
 
 # To get the additional information of the Pod in the namespace "apps"
-$ kubectl get po -n apps -owide
+$ kubectl get po -n apps -o wide
```

Comment:
It will work with **-owide** as well so no need to specify space between -o and wide. 